src/Morphic-Base/MouseClickState.class.st

```diff
@@ -174,7 +174,10 @@ MouseClickState >> doubleClickTimeoutSelector: anObject [
 { #category : 'event handling' }
 MouseClickState >> drag: event [
 
-	dragSelector ifNotNil: [clickClient perform: dragSelector with: event]
+	dragSelector ifNotNil: [
+		clickClient perform: dragSelector with: event ].
+	WorldMorph cavroisDragBlock ifNotNil: [
```

Comment:
What about 

WorldMorph executeDragBlockWith: clickClient

like this we do not expose the ifNotNil....